quobyte.go

```diff
@@ -168,3 +179,23 @@ func (client *QuobyteClient) SetTenant(tenantName string) (string, error) {
 
 	return response.TenantID, nil
 }
+
+// IsValidUUID Validates given uuid
+func IsValidUUID(uuid string) bool {
+	r := regexp.MustCompile("^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-4[a-fA-F0-9]{3}-[8|9|aA|bB][a-fA-F0-9]{3}-[a-fA-F0-9]{12}$")
+	return r.MatchString(uuid)
+}
+
+// ResolveTeantanNameTOUUID Returns UUID for given name, error if not found.
+func (client *QuobyteClient) ResolveTeantanNameTOUUID(name string) (string, error) {
```

Comment:
Nit: Typo, probably should be 'ResolveTenantNameToUuid' or 'ResolveTenantNameTOUUID'